app/components/pages-menu/component.js

```diff
@@ -4,6 +4,65 @@ export default Ember.Component.extend({
     style: Ember.computed('layer.settings.values.color', function(){
         return Ember.String.htmlSafe('color: ' + (this.get('layer.settings.values.color') || '#333') + '; ');
     }),
+    sticky: Ember.observer('layer.settings.values.stickToTop' , 'layers.[]' , function() {
+        let topOfNav = null;
+        if( $('.pages-menu')[0] ){
+            topOfNav = $('.pages-menu').offset().top;
+        }
+        let self = this;
+        if(this.get('layer.settings.values.stickToTop')){
+            $(window).on('scroll.nav', function (e) {
+
+
+                let _topOfNav = null;
```

Comment:
I wouldn't use underscore here, it means something for other languages, you should name it something else, i.e. paddedNavOffset